app/src/main/java/com/example/calculator/MainActivity.kt

```diff
@@ -0,0 +1,143 @@
+package com.example.calculator
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity(), View.OnClickListener {
+    private var string: String = "0"
+    private var soA: Double = 0.0
```

Comment:
```suggestion
    private var firstNumber = 0.0
```
Đặt tên bằng tiếng anh hết nhé